frameworks/PHP/limonade/deploy/nginx.conf

```diff
@@ -0,0 +1,46 @@
+worker_processes  8;
+
+error_log stderr error;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       /home/knewman/FrameworkBenchmarks/installs/nginx/conf/mime.types;
```

Comment:
Also, probably need to change `/home/knewman/FrameworkBenchmarks/installs/nginx/` to `/usr/local/nginx/` for the sed in setup.sh to work (unless you change the setup.sh file).